test/functional/custom-transform.spec.ts

```diff
@@ -66,62 +125,51 @@ describe("custom transformation decorator", () => {
 
     });
 
-    it("versions and groups should work with @Transform decorator too", () => {
+    it("Non class objects should not break if transform is used", () => {
```

Comment:
When I run all tests I get the below outcome. Adding the class transformer in the one test breaks other tests.

`  229 passing (8s)
  62 failing

  1) action parameters @Body should provide a json object for json-typed controllers and actions asserting port 3001:
     AssertionError: expected status code 500 to equal 200
`